onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -811,6 +813,11 @@ def _where_input_wrangler(
         matcher=lambda sample: sample.kwargs.get("rounding_mode") is not None,
         reason="rounding_mode is not yet supported",
     ),
+    skip(
+        "grid_sampler_2d",
+        matcher=lambda sample: sample.args[1] == 2,
+        reason="'bicubic' mode in ORT implemented differently with Torch",
```

Comment:
```suggestion
        reason="fixme: 'bicubic' mode in ORT implemented differently with Torch",
```

Reply:
Torch implemented this using the cubic convolution algorithm with alhpa=−0.75, might be different than ORT. So it maybe by design.